dht_bootstrap.go

```diff
@@ -117,60 +95,42 @@ func (dht *IpfsDHT) bootstrapBuckets(ctx context.Context) error {
 	}
 
 	buckets := dht.routingTable.GetAllBuckets()
-	var wg sync.WaitGroup
-	errChan := make(chan error)
-
+	if len(buckets) > 16 {
```

Comment:
👍

Reply:
Should we exclude the last bucket? (catch-all bucket)

Reply:
https://github.com/libp2p/go-libp2p-kad-dht/issues/406

We should include this bucket, but we should also be including more buckets (maybe, sometimes).